test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -682,6 +682,41 @@ public async Task UpdatePackageForPR_PackageNamespace_WithMultipleFeedsWithIdent
             solutionService.Save();
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
+        public async Task VerifyCatchFileInsideObjFolder(ProjectTemplate projectTemplate)
+        {
+            using (var testContext = new ApexTestContext(VisualStudio, projectTemplate, XunitLogger))
+            {
+                //Arrange
+                var packageName = "TestPackage";
+                var packageVersion = "1.0.0";
+                await CommonUtility.CreatePackageInSourceAsync(testContext.PackageSource, packageName, packageVersion);
+                var project = testContext.SolutionService.AddProject(ProjectLanguage.CSharp, ProjectTemplate.NetCoreConsoleApp, "NetCore");
+                var solutionService = VisualStudio.Get<SolutionService>();
+                var nugetConsole = GetConsole(testContext.Project);
+                nugetConsole.InstallPackageFromPMC(packageName, packageVersion);
+                FileInfo CatchFilePath = CommonUtility.GetCatchFilePath(project.FullPath);
+
+                //Act
+                testContext.SolutionService.Build();
+                testContext.NuGetApexTestService.WaitForAutoRestore();
+                CommonUtility.WaitForFileExists(CatchFilePath);
+
+                project.Rebuild();
+                CommonUtility.WaitForFileExists(CatchFilePath);
+
+                project.Clean();
+                CommonUtility.WaitForFileNotExists(CatchFilePath);
+
+                nugetConsole.Execute("dotnet restore");
+                CommonUtility.WaitForFileExists(CatchFilePath);
+
+                nugetConsole.Clear();
```

Comment:
The last 2 lines are probably unnecessary as Save is happening through Dispose of the ApexTestContext..

Reply:
OK, we can delete the last 2 lines. Thanks.